synchronization/src/main/java/de/cyface/synchronization/WiFiSurveyor.java

```diff
@@ -429,20 +435,41 @@ public boolean isPeriodicSyncEnabled() {
      * <b>Attention:</b>
      * If you want to check if periodic ("auto") sync is enabled which is automatically set when the network state
      * changes, see {@link #isPeriodicSyncEnabled()}.
+     * <p>
+     * This method must be called after {@link #startSurveillance(Account)} is called.
      *
      * @return True if synchronization is enabled
      */
+    @SuppressWarnings("unused") // SDK implementing apps may check weather sync is disabled completely
     public boolean isSyncEnabled() {
         return ContentResolver.getIsSyncable(currentSynchronizationAccount, authority) == 1;
     }
 
     /**
      * Allows to enable or disable synchronization completely.
+     * <p>
+     * This method must be called after {@link #startSurveillance(Account)} is called. You can also use the
+     * {@link #setSyncEnabled(Account, boolean)} which does not have this requirement.
      *
      * @param enabled True if synchronization should be enabled
      */
+    @SuppressWarnings("unused") // SDK implementing apps may want to disable synchronization completely
     public void setSyncEnabled(final boolean enabled) {
```

Comment:
the old interface was not changed, I just added a new one to activate accounts which are not yet linked in the surveyor (only after startSurveillance is called it's linked).

Thus, no API upgrade in the code version.